happner is broken on Windows

Problem is path handling with hardcoded separators. Please use path.join() or path.sep!

```
[FATAL] - Error: Cannot find module 'E:\git\open-sdk-sample-app\node_modules\happn\lib\index.js/package.json'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at new Packager (E:\git\open-sdk-sample-app\node_modules\happner\lib\system\packager.js:37:19)
    at Mesh._initializePackager (E:\git\open-sdk-sample-app\node_modules\happner\lib\mesh.js:951:18)
    at async.series.isServer (E:\git\open-sdk-sample-app\node_modules\happner\lib\mesh.js:526:17)
    at E:\git\open-sdk-sample-app\node_modules\happner\node_modules\async\lib\async.js:607:21
    at E:\git\open-sdk-sample-app\node_modules\happner\node_modules\async\lib\async.js:246:17
    at iterate (E:\git\open-sdk-sample-app\node_modules\happner\node_modules\async\lib\async.js:146:13)
    at E:\git\open-sdk-sample-app\node_modules\happner\node_modules\async\lib\async.js:157:25
    at E:\git\open-sdk-sample-app\node_modules\happner\node_modules\async\lib\async.js:248:21
    at E:\git\open-sdk-sample-app\node_modules\happner\node_modules\async\lib\async.js:612:34
    at E:\git\open-sdk-sample-app\node_modules\happner\lib\mesh.js:1606:7
    at PubSubService.handleDataResponseLocal (E:\git\open-sdk-sample-app\node_modules\happn\lib\services\pubsub\service.
js:327:12)
    at E:\git\open-sdk-sample-app\node_modules\happn\lib\client\plugins\intra-process.js:140:33
[ INFO] - 22ms  home (Mesh) exit 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,5 @@
 os:
   - linux
-  - osx
 
 language: node_js
 node_js:

--- a/RELEASES.md
+++ b/RELEASES.md
@@ -395,3 +395,5 @@
 -----------------
 
 - fix for path sep in packager
+- added appveypr.yml
+- removed osx tests on travis

--- a/RELEASES.md
+++ b/RELEASES.md
@@ -390,3 +390,8 @@
 -----------------
 
 - updated to happn 2.15.8
+
+1.22.3 2016-10-17
+-----------------
+
+- fix for path sep in packager

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,0 +1,21 @@
+# Test against this version of Node.js
+environment:
+  nodejs_version: "0.10"
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js or io.js
+  - ps: Install-Product node $env:nodejs_version
+  # install modules
+  - npm install
+
+# Post-install test scripts.
+test_script:
+  # Output useful info for debugging.
+  - node --version
+  - npm --version
+  # run tests
+  - npm test
+
+# Don't actually build.
+build: off

--- a/lib/system/packager.js
+++ b/lib/system/packager.js
@@ -20,6 +20,7 @@ var fs = require('fs')
   , minifyJS = require('uglify-js').minify
   , normalize = require('path').normalize
   , dirname = require('path').dirname
+  , sep = require('path').sep
   , happnProtocol
   ;
 
@@ -30,18 +31,18 @@ function Packager(mesh) {
   this.log = mesh.log.createLogger('Packager');
   mesh.tools.packages = this.packages = {};
 
-  var script1 = dirname(dirname(require.resolve('bluebird'))) + '/browser/bluebird.js';
+  var script1 = dirname(dirname(require.resolve('bluebird'))) +  sep + 'browser' + sep + 'bluebird.js';
   var script2 = normalize(mesh._mesh.datalayer.server.services.pubsub.script);
-  var script3 = dirname(require.resolve('happn')) + '/client/base.js';
+  var script3 = dirname(require.resolve('happn')) + sep + 'client' + sep + 'base.js';
 
-  happnProtocol = require(require.resolve('happn').replace('/lib/index.js', '') + '/package.json').protocol;
+  happnProtocol = require(require.resolve('happn').replace(sep + 'lib' + sep + 'index.js', '') + sep + 'package.json').protocol;
 
-  var script4 = normalize(__dirname + '/shared/logger.js');
-  var script5 = normalize(__dirname + '/shared/promisify.js');
-  var script6 = normalize(__dirname + '/shared/mesh-error.js');
-  var script7 = normalize(__dirname + '/shared/messenger.js');
-  var script8 = normalize(__dirname + '/shared/internals.js');
-  var script9 = normalize(__dirname + '/shared/mesh-client.js');
+  var script4 = normalize(__dirname + sep + 'shared' + sep + 'logger.js');
+  var script5 = normalize(__dirname + sep + 'shared' + sep + 'promisify.js');
+  var script6 = normalize(__dirname + sep + 'shared' + sep + 'mesh-error.js');
+  var script7 = normalize(__dirname + sep + 'shared' + sep + 'messenger.js');
+  var script8 = normalize(__dirname + sep + 'shared' + sep + 'internals.js');
+  var script9 = normalize(__dirname + sep + 'shared' + sep + 'mesh-client.js');
 
   this.scripts = [
     {watch: false, min: false, file: script1},
@@ -52,7 +53,7 @@ function Packager(mesh) {
     {watch: true, min: false, file: script6},
     {watch: true, min: false, file: script7},
     {watch: true, min: false, file: script8},
-    {watch: true, min: false, file: script9},
+    {watch: true, min: false, file: script9}
     //
     // When not running NODE_ENV=production the package is re-assembled
     // when this file changes.
@@ -196,9 +197,7 @@ Packager.prototype.assemble = function () {
 
       var scriptStr = script.buf.toString();
 
-      if (script.file.indexOf('client/base.js') > -1)
-        scriptStr = scriptStr.replace('{{protocol}}', happnProtocol);
-
+      if (script.file.indexOf('client' + sep + 'base.js') > -1) scriptStr = scriptStr.replace('{{protocol}}', happnProtocol);
       _this.merged.script += scriptStr + '\n';
     });
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "happner",
-  "version": "1.22.2",
+  "version": "1.22.3",
   "description": "distributed application engine with evented storage and mesh services",
   "main": "lib/mesh.js",
   "bin": {

--- a/test/e3a-rest-component.js
+++ b/test/e3a-rest-component.js
@@ -89,7 +89,7 @@ describe('e3a-rest-component', function () {
 
     var timedOut = setTimeout(function(){
       callback(new Error('remote mesh start timed out'));
-    },5000);
+    }, 60000);
 
     // spawn remote mesh in another process
     remote = spawn('node', [libFolder + REMOTE_MESH]);
@@ -102,7 +102,7 @@ describe('e3a-rest-component', function () {
 
         setTimeout(function(){
           callback();
-        },1000);
+        }, 1000);
       }
     });
   };

